doc/docs/2.3.x/how-tos/jupyterlab-extension/index.md

```diff
@@ -199,102 +141,147 @@ Replace the image name with your own image otherwise.
+4. Find the **IP address** you used to access the JupyterHub as described in these [Helm installation instructions](https://zero-to-jupyterhub.readthedocs.io/en/latest/jupyterhub#setup-jupyterhub) (Step 5 and 6) and open Jupyterlab.
+5. Click on the link provided in the `stdout` of your terminal to run JupyterLab in a browser.
+6. Connect to your cluster using the `grpc://<cluster-ip>:<port>` format.
+7. Run the following command to refresh the mount server:
 
     ``` shell
     umount /pfs
     ```
 
+##### Automate Cluster Details
+
+You can specify your `pachd` cluster details in your Helm chart via `extraFiles` to avoid having to provide them every time Jupyter Hub starts. The `mountPath` input is required, however the location does not matter.
+
+```yaml
+singleuser:
+    defaultUrl: "/lab"
+    image:
+        name: pachyderm/notebooks-user
+        tag: <latest-release>
+    extraEnv:
+        "SIDECAR_MODE": "True"
+    extraContainers:
+        - name: mount-server-manager
+          image: pachyderm/mount-server:<latest-release> 
+          command: ["/bin/bash"]
+          args: ["-c", "mkdir -p ~/.pachyderm && cp /config/config.json ~/.pachyderm && mount-server"]
+          volumeMounts:
+              - name: shared-pfs
+                mountPath: /pfs
+                mountPropagation: Bidirectional
+              - name: files
+                mountPath: /config
+          securityContext:
+              privileged: true
+              runAsUser: 0
+    storage:
+        extraVolumeMounts:
+            - name: shared-pfs
+              mountPath: /pfs
+              mountPropagation: HostToContainer
+        extraVolumes:
+            - name: shared-pfs
+              emptyDir: {}
+    extraFiles:
+      config.json:
+        mountPath: </any/path/file.json>
+        data:
+          v2:
+            active_context: mount-server
+            contexts:
+              mount-server:
+                source: 2
+                pachd_address: <cluster_endpoint>
+                server_cas: <b64e_cert_string>
+                session_token: <token>
+            metrics: true
+```
+
+
+<!--
+// the following can be used if you do not know your cluster IP address (production)
+```
+docker run -it -v ~/.pachyderm/config.json:/home/jovyan/.pachyderm/config.json -p 8888:8888 -e GRANT_SUDO=yes --user root --device /dev/fuse --privileged --entrypoint /opt/conda/bin/jupyter pachyderm/notebooks-user:v0.6.0 lab --allow-root
+``` 
+-->
+
+--- 
+
+## How to Use 
+
+### Mount a Branch
+
+1. Open the Jupyterlab UI.
+2. Open a **Terminal** from the launcher.
+3. Navigate to the **Mounted Repositories** tab.
+4. Input the following to see a demo repo appear:
+    ```sh 
+    pachctl create repo demo
+    pachctl create branch demo@master
+    ```
+5. Scroll to the **Unmounted Repositories** section.
+6. Select **Mount** next to the **Demo** repository. 
+7. Input the following to create a simple text file:
```

Comment:
When we mount a repo, it mounts a specific commit. As a result, after this `put file`, a new commit will be created on the branch. However, what's still mounted is the old commit. Users will see next to their repo a notification saying that they're currently `1 commit behind`. Unmounting and re-mounting the same repo will then allow users to `cat /pfs/demo/myfile.txt`.